src/dioptra/restapi/experiment/service.py

```diff
@@ -130,9 +130,28 @@ def rename_experiment(
     @staticmethod
     def get_all(**kwargs) -> List[Experiment]:
         log: BoundLogger = kwargs.get("log", LOGGER.new())  # noqa: F841
-
         return Experiment.query.filter_by(is_deleted=False).all()  # type: ignore
 
+    @staticmethod
```

Comment:
`staticmethod` can be removed, and `self` added to the `get_page` to conform to the recent changes for the Restapi.